src/lib/cache/subCache.cpp

```diff
@@ -71,8 +71,8 @@ volatile SubCacheState subCacheState = ScsIdle;
 */
 EntityInfo::EntityInfo(const std::string& _entityId, const std::string& _entityType, const std::string& _isPattern)
 {
-  entityId     = _entityId;
-  entityType   = _entityType;
+  const std::string entityId     = _entityId;
+  const std::string entityType   = _entityType;
```

Comment:
What exactly did cppcheck complain about here?
We'll need a new way to get rid of the warnings ...